pages/content/amp-dev/documentation/components/reference/amp-bind@ar.md

```diff
@@ -180,7 +180,7 @@ limitations under the License.
 
 # تعديل الحالة باستخدام `AMP.setState()`
 
-يُدمِج الإجراء [`AMP.setState()`](../../spec/amp-actions-and-ents.md#amp) كائنًا حرفيًا في الحالة. عند الضغط على الزر أدناه مثلاً، سيعمل `AMP.setState()` على [الدمج العميق](#deep-merge-with-ampsetstate) للكائن الحرفي مع الحالة.
+يُدمِج الإجراء [`AMP.setState()`](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-ents.md#amp) كائنًا حرفيًا في الحالة. عند الضغط على الزر أدناه مثلاً، سيعمل `AMP.setState()` على [الدمج العميق](#deep-merge-with-ampsetstate) للكائن الحرفي مع الحالة.
```

Comment:
this should link to the guide on amp.dev

Reply:
I changed all links to imported files and also created an issue for the imported files: https://github.com/ampproject/amp.dev/issues/2645